src/libs/actions/PersonalDetails.js

```diff
@@ -114,7 +114,8 @@ function fetchTimezone() {
         .then((data) => {
             const timezone = lodashGet(data, 'nameValuePairs.timeZone.selected', 'America/Los_Angeles');
             Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, {timezone});
-        });
+        })
+        .catch(error => console.debug('Error fetching user timezone', error));
```

Comment:
Why is this being added?

Reply:
It was also throwing errors when we were offline so I thought to catch it. Let me know, I will rollback this.

Reply:
Ok, no problem it's fine to leave like this for now.